src/palace/manager/integration/license/overdrive/api.py

```diff
@@ -589,6 +611,138 @@ def _get_book_list_page(
         availability_queue = extractor_class.availability_link_list(content_dict)
         return availability_queue, next_link
 
+    def _get_headers(self, auth_token: str) -> dict[str, str]:
+        return {"Authorization": f"Bearer {auth_token}"}
+
+    def book_info_initial_endpoint(
+        self,
+        start: datetime.datetime | None = None,
+        page_size: int = PAGE_SIZE_LIMIT,
+    ) -> BookInfoEndpoint:
+        """Create an initial book info url."""
+
+        # if no start date specified, assume effect beginning of time.
+        if not start:
+            start = datetime.datetime(1970, 1, 1, 0, 0, 0, tzinfo=datetime.UTC)
+        last_update_time = start - self.EVENT_DELAY
+        self.log.info("Creating url for circulation changes since %s", last_update_time)
+        last_update = last_update_time.strftime(self.TIME_FORMAT)
+
+        book_info_initial_endpoint = self.endpoint(
+            self.EVENTS_ENDPOINT,
+            # From https://developer.overdrive.com/apis/search:
+            # "**Note: When you search using the lastTitleUpdateTime or
+            # lastUpdateTime parameters, your results will be automatically
+            # sorted in ascending order (and all other sort options will be ignored)."
+            lastupdatetime=last_update,
+            limit=str(min(page_size, self.PAGE_SIZE_LIMIT)),
+            collection_token=self.collection_token,
+        )
+        endpoint: str = _make_link_safe(book_info_initial_endpoint)
+
+        return BookInfoEndpoint(endpoint)
+
+    async def fetch_book_info_list(
+        self,
+        endpoint: BookInfoEndpoint,
+        rel_to_follow: str = "next",
```

Comment:
Minor: Is there ever a case where we would change this? If not does it make sense to have this as a constant rather then a parameter?